uws/UWS/models.py

```diff
@@ -10,6 +10,25 @@ def _parse_bool(self, value):
             return False
         return value
 
+    def _find_ns_uws(self, namespaces):
+        # Search for namespace that contains "http://www.ivoa.net/xml/UWS/"
+        # (usually, there is a version attached, which we ignore here),
+        # for using it (including {}) as uws_ns in the
+        # find/findall-functions everywhere below.
+        ns = None
+        for p in namespaces.keys():
+            value = namespaces[p]
+            if 'http://www.ivoa.net/xml/UWS/' in value:
+                ns = value
```

Comment:
I have seen below why you need the brackets. Maybe you can add a `_findall` and `_find` method to base, that will do all the xml searching for you, then we could also get rid of much of the repetition. The `uws_namespace` could be saved into the model as a property, then you don't need to pass it to these functions:

`def _findall(self, xml_node, node_name, parsed)`
and in here you can do all the curly bracket adding and everything else.

and something similar for find. However for the `xml_nodes` something else would be needed (either findall takes parsed or xml_nodes and acts accordingly (easy solution) or you determine the class type through `isinstance` (not sure this works though)).

What do you think?